doc/api/next_api_changes/removals/20314-GL.rst

```diff
@@ -0,0 +1,11 @@
+Colorbar related removals
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``Colorbar.on_mappable_changed()`` and ``Colorbar.update_bruteforce()`` have
+been removed, you can use ``Colorbar.update_mappable()`` instead.
```

Comment:
Isn't it `update_normal`?

Reply:
Yes, good catch. It is updating the entire mappable, but called `update_normal`... No wonder I got confused :)